tests/Integration/Behaviour/Features/Context/Domain/Product/Combination/UpdateCombinationStockFeatureContext.php

```diff
@@ -74,7 +74,9 @@ public function transformCombinationStock(TableNode $tableNode): CombinationStoc
             (int) $dataRows['low stock threshold'],
             PrimitiveUtils::castStringBooleanIntoBoolean($dataRows['low stock alert is enabled']),
             $dataRows['location'],
-            '' === $dataRows['available date'] ? null : new DateTime($dataRows['available date'])
+            '' === $dataRows['available date'] ? null : new DateTime($dataRows['available date']),
+            [],
+            []
```

Comment:
thats not how you test a new feature :smile: 
Im afraid you will have to modify much more than that and add related scenrios to actually test it all works as expected.

Reply:
@zuk3975 Yes, that was just to make tests work. 😄 I will maybe need some help on that 🙏

PS: Still better then deleting the test 🤣

Reply:
done